interfaces/builtin/uefi_manager.go

```diff
@@ -0,0 +1,190 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"bytes"
+	"github.com/snapcore/snapd/interfaces"
+)
+
+var uefiManagerPermanentSlotAppArmor = []byte(`
+# Description: Allow operating as the uefiManager service. Reserved because this
+#  gives privileged access to the system.
+# Usage: reserved
+
+  capability sys_admin,
+  capability linux_immutable,
+
+  # File accesses
+  /sys/firmware/efi/                    r,
+  /sys/firmware/efi/fw_platform_size    r,
+  /sys/firmware/efi/esrt/entries/       r,
+  /sys/firmware/efi/esrt/entries/**     r,
+  /sys/firmware/efi/efivars/            r,
+  /sys/firmware/efi/efivars/**          rw,
+  /sys/devices/virtual/dmi/             r,
+  /sys/devices/virtual/dmi/**           r,
+  /usr/bin/gpg                          ix,
+  /boot/efi/EFI/ubuntu/fw/              rw,
+  /boot/efi/EFI/ubuntu/fw/**            rw,
+  @{PROC}/@{pid}/mounts                 r,
+  /sys/devices/**/partition             r,
+  /dev/sd[a-z]*                         r,
```

Comment:
I suspect those mount parts are needed to read the UEFI data somewhere. Can this be commented a bit more so its clear what those things are needed for?